app/gateway/src/Types/API/Search.hs

```diff
@@ -38,8 +39,16 @@ newtype SearchReq = SearchReq
 
 type OnSearchReq = CallbackReq Value
 
+newtype RawHeader = RawHeader {getRawHeader :: ByteString}
+
+instance FromHttpApiData RawHeader where
+  parseUrlPiece = parseHeader . DT.encodeUtf8
+  parseQueryParam = parseUrlPiece
+  parseHeader = Right . RawHeader
+
 type SearchAPI =
-  SignatureAuth "Authorization"
+  Header "Authorization" RawHeader
```

Comment:
Can you add a comment, that these headers are added by middleware.
Just to avoid confusion in the future